core/include/thread.h

```diff
@@ -260,13 +260,15 @@ struct _thread {
  *       is considered to be a bad programming practice and we strongly
  *       discourage you from doing so.
  *
+ * @pre The @p name parameter must stay valid even after the thread has terminated (e.g. using a literal).
```

Comment:
```suggestion
 * @pre The @p name parameter must stay valid even after the thread has
 *      terminated (e.g. using a literal).
```

To make the static test happy :)

Reply:
Thanks. I've taken the liberty to autosquash after pushing the fixup.